fixture.go

```diff
@@ -42,13 +42,15 @@ type Fixture struct {
 
 	Home Competitor `json:"home"`
 	Away Competitor `json:"away"`
+
+	ExtraInfo []ExtraInfo `xml:"extra_info>info,omitempty" json:"extraInfo,omitempty"`
+	Raw       []byte      // fixture API raw XML response; keep for downstream processing
```

Comment:
U Message vec postoji Raw atribut namjenjen za ovu svrhu. Iskoristi njega. 
Razumijem da ti on u ovom trenutku nije dostupan u api i da ti je ovako lakse. No to su mi premali argumenti da bi sijao identicne atribute. Zbunjivao buduce korisnike.

Reply:
> U Message vec postoji Raw atribut namjenjen za ovu svrhu. Iskoristi njega.
> Razumijem da ti on u ovom trenutku nije dostupan u api i da ti je ovako lakse. No to su mi premali argumenti da bi sijao identicne atribute. Zbunjivao buduce korisnike.

Inicijalno sam to htio ovako riješiti:
```
type fixtureAPI interface {
	Fixture(lang uof.Lang, eventURN uof.URN) ([]byte, error)
	Fixtures(lang uof.Lang, to time.Time) (<-chan uof.Fixture, <-chan error)
}
```

I složiti da se  poziva `NewAPIMessage` za fixture umjesto `NewFixtureMessage` u pipe/fixture.go. Ili promijeniti NewFixtureMessage da prima `[]byte` umjseto `Fixture`

api/sports.go
```
// Fixture lists the fixture for a specified sport event
func (a *API) Fixture(lang uof.Lang, eventURN uof.URN) ([]byte, error) {
	buf, err := a.get(pathFixture, &params{Lang: lang, EventURN: eventURN})
	if err != nil {
		return nil, err
	}
	return buf, err
}
```